guides/release/pages.yml

```
@@ -1,325 +1,325 @@
-    - title: "Component Arguments and HTML Attributes"
+    - title: "Arguments des Composants et Attributs HTML"
       url: "component-arguments-and-html-attributes"
-    - title: "Conditional Content"
+    - title: "Contenu Conditionnel"
       url: "conditional-content"
-    - title: "Block Content"
+    - title: "Contenu des Blocs"
       url: "block-content"
-    - title: "Helper Functions"
+    - title: "Fonctions Helper"
       url: "helper-functions"
-    - title: "Component State and Actions"
+    - title: "État des Composants et Actions"
       url: "component-state-and-actions"
-    - title: "Looping Through Lists"
+    - title: "Boucler sur des Listes"
       url: "looping-through-lists"
-    - title: Template Lifecycle, DOM, and Modifiers
+    - title: Cycle de Vie des Templates, DOM, et Modifiers
       url: template-lifecycle-dom-and-modifiers
-    - title: "Built-in Components"
+    - title: "Composants Built-in"
       url: "built-in-components"
 - title: "Routing"
   url: "routing"
   pages:
     - title: "Introduction"
       url: "index"
-    - title: "Defining Your Routes"
+    - title: "Définir une Route"
       url: "defining-your-routes"
-    - title: "Linking Between Routes"
+    - title: "Liens entre les Routes"
       url: "linking-between-routes"
-    - title: "Specifying a Route's Model"
+    - title: "Spécifier le Modèle d'une Route"
       url: "specifying-a-routes-model"
-    - title: "Rendering a Template"
+    - title: "Rendre un Template"
       url: "rendering-a-template"
-    - title: "Redirecting"
+    - title: "Redirections"
       url: "redirection"
-    - title: "Preventing and Retrying Transitions"
+    - title: "Empêcher et Réessayer une Transition"
       url: "preventing-and-retrying-transitions"
       isAdvanced: true
-    - title: "Loading / Error Substates"
+    - title: "Chargement / Sous-États d'Erreur"
       url: "loading-and-error-substates"
       isAdvanced: true
-    - title: "Query Parameters"
+    - title: "Query Params"
       url: "query-params"
-    - title: "Asynchronous Routing"
+    - title: "Routing Asynchrone"
       url: "asynchronous-routing"
       isAdvanced: true
-    - title: "Controllers"
+    - title: "Contrôleurs"
       url: "controllers"
 - title: "Services"
   url: "services"
   pages:
-    - title: Overview
+    - title: Aperçu
       url: "index"
 - title: "Ember Data"
   url: "models"
   pages:
     - title: "Introduction"
       url: "index"
-    - title: "Defining Models"
+    - title: "Définir des Modèles"
       url: "defining-models"
-    - title: "Finding Records"
+    - title: "Requêter des Records"
       url: "finding-records"
-    - title: "Creating, Updating and Deleting"
+    - title: "Créer, Mettre à Jour et Supprimer"
       url: "creating-updating-and-deleting-records"
     - title: "Relationships"
       url: "relationships"
-    - title: "Pushing Records into the Store"
+    - title: "Pousser des Records dans le Store"
```

`Pousser` sounds weird to me.

```suggestion
    - title: "Stocker des Records dans le Store"
```